packages/jest-util/src/index.js

```diff
@@ -47,5 +46,4 @@ module.exports = {
   installCommonGlobals,
   isInteractive,
   setGlobal,
-  validateCLIOptions,
```

Comment:
technically a breaking change. not sure if it matters, though

Reply:
It absolutely matters.  Please see https://github.com/facebook/jest/issues/5749